.github/workflows/ci.yml

```diff
@@ -3,6 +3,7 @@ name: CI
 on:
   push:
     branches: [ master ]
+    tags: [ "**" ]
```

Comment:
why double ? just asking :)

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
> `*`: Matches zero or more characters, but does not match the / character. For example, Octo* matches Octocat.
> `**`: Matches zero or more of any character.

So it will match even if the tag ends up having a `/` in it.  Not something we need, just my default at this point when saying 'anything'.